test-cluster-without-topology.mk

```diff
@@ -2,11 +2,11 @@ TEST_NAMESPACE=ns-without-topology
 TEST_CLUSTER_NAME=mycluster-without-topology
 
 .PHONY: test-cluster-create
-test-cluster-create: $(CLUSTERCTL) ## Create a workload cluster without topology
-	$(CLUSTERCTL) version
-	$(CLUSTERCTL) config repositories | grep nutanix
-	$(CLUSTERCTL) generate cluster ${TEST_CLUSTER_NAME} -i nutanix:${LOCAL_PROVIDER_VERSION} --list-variables -v 10
-	$(CLUSTERCTL) generate cluster ${TEST_CLUSTER_NAME} -i nutanix:${LOCAL_PROVIDER_VERSION} --target-namespace ${TEST_NAMESPACE}  -v 10 > ./${TEST_CLUSTER_NAME}.yaml
+test-cluster-create: ## Create a workload cluster without topology
```

Comment:
why do we need this change - $(CLUSTERCTL) => clusterctl?

Reply:
$(CLUSTERCTL) was previously a variable defined in the makefile which pointed to a path 

```
CLUSTERCTL := $(abspath $(TOOLS_BIN_DIR)/$(CLUSTERCTL_BIN))  
```

since we are using devbox and all of the binaries are added to the path we no longer need this variable